### PR TITLE
f-mega-modal@0.8.0 and 4 other packages - remove styles which should come from fozzie

### PR DESCRIPTION
**Pull request by dandel10n**
### Removed
- `u-overlay` css style definition as should come from `fozzie`

### Removed
- `is-visuallyHidden` css style definition as should come from `fozzie`

### Removed
- `is-visuallyHidden` css style definition as should come from `fozzie`

### Removed
- `is-visuallyHidden`, `is-hidden`, `is-hidden--noJS`, `is-shown`, `is-shown--noJS` css style definition as should come from `fozzie`

### Added
- Focus styles for navigation links for transparent and orange header versions

### Removed
- `is-visuallyHidden` css style definition as should come from `fozzie`

### CODE REVIEW

File: packages/components/molecules/f-mega-modal/src/components/MegaModal.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <template>
     <div
         ref="megaModal"
-        :class="['c-megaModal', $style['c-megaModal'], {
-            [$style['u-overlay']]: showOverlay
-        }]"
+        :class="['c-megaModal',
+                 $style['c-megaModal'],
+                 { 'u-overlay': showOverlay }]"
```
Comment (DamianMullins) [inline packages/components/molecules/f-mega-modal/src/components/MegaModal.vue:6]: Looks like the previous change here has been reverted?

File: packages/components/molecules/f-mega-modal/src/components/MegaModal.vue [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <template>
     <div
         ref="megaModal"
-        :class="['c-megaModal', $style['c-megaModal'], {
-            [$style['u-overlay']]: showOverlay
-        }]"
+        :class="['c-megaModal',
+                 $style['c-megaModal'],
```
Comment (DamianMullins) [inline packages/components/molecules/f-mega-modal/src/components/MegaModal.vue:5]: NIT: Do you need change the line-breaks here?

Comment (dandel10n) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue:5]: eslint fixed it this way

Comment (DamianMullins) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue:5]: Weird, the style used elsewhere in the template is different.. Never mind though.

**Review by DamianMullins — DISMISSED** [review]

**Review by ashleynolan — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Review by DamianMullins — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**